core/src/main/scala/precog/SbtPrecogBase.scala

```diff
@@ -131,24 +146,34 @@ abstract class SbtPrecogBase extends AutoPlugin {
         "-Ywarn-dead-code",
         "-Xfuture")
 
+    val scalafmtSettings: Seq[Def.Setting[_]] = Seq(
```

Comment:
`.scalafmt-common.conf` will always be copied to the project that uses `sbt-precog` (i.e. it will override a file that's already there). Projects should never check in `.scalafmt-common.conf` and should always add it to `.gitignore`.

`.scalafmt.conf` will only be copied if it is not there yet and can be used in case a setting needs to be changed. It only needs to be checked in if it is changed.